config-builder/src/test/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/EncassEntityBuilderTest.java

```diff
@@ -101,7 +101,7 @@ private static void buildBundleWithEncass(Bundle bundle, BundleType deployment,
         assertNotNull(getSingleChildElement(xml, NAME));
         assertEquals(encassName, getSingleChildElementTextContent(xml, NAME));
         assertNotNull(getSingleChildElement(xml, GUID));
-        assertEquals(EncassEntityBuilderTest.TEST_GUID, getSingleChildElementTextContent(xml, GUID));
+        //assertEquals(EncassEntityBuilderTest.TEST_GUID, getSingleChildElementTextContent(xml, GUID));
```

Comment:
Why is this commented?

Reply:
entity id will not be same, it will be regenerated if its not reusable